lm75bd/lm75bd.c

```diff
@@ -26,8 +27,31 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  error_code_t errCode;
+
+  /* Selecting temperature register */
+  uint8_t sendData = 0;
+  errCode = i2cSendTo(devAddr, &sendData, 1);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
+
+  /* Receive temp reading */
+  uint8_t buff[2] = {0, 0};
```

Comment:
It's easier to initialize a buffer to 0 by doing this.
```suggestion
  uint8_t buff[2] = {0};
```

If a buffer is partially initialized, the rest of the elements will be initialized to 0.